apps/consulting/pages/nebari-services.tsx

```diff
@@ -399,7 +396,7 @@ export const NebariServicesPage: FC<TContainerProps> = ({
         >
           <SwiperSlide>
             <figure className="py-10 px-24 text-white">
-              <blockquote className="before:relative before:top-[45px] before:left-[-30px] text-[1.8rem] before:text-[10rem] italic font-semibold leading-[2.7rem] before:text-white before:content-['“']">
+              <blockquote className="before:relative before:top-[36px] before:left-[-36px] text-[1.8rem] before:text-[10rem] italic font-semibold leading-[2.7rem] before:text-white before:content-['“']">
```

Comment:
The decorative (top, left) quotation mark in the testimonials was not quite aligned correctly (as in the Figma file)